node/coinstacks/solana/api/src/controller.ts

```diff
@@ -9,24 +9,30 @@ import {
   ValidationError,
   handleError,
 } from '../../../common/api/src' // unable to import models from a module with tsoa
-import { Account, PriorityFees, TxHistory } from './models'
+import { Account, PriorityFees, RawTx, Tx, TxHistory } from './models'
 import { Helius } from 'helius-sdk'
+import axios from 'axios'
 
 const RPC_URL = process.env.RPC_URL
 const RPC_API_KEY = process.env.RPC_API_KEY
+const INDEXER_URL = process.env.INDEXER_URL
 
 const NETWORK = process.env.NETWORK
 
 if (!NETWORK) throw new Error('NETWORK env var not set')
 if (!RPC_URL) throw new Error('RPC_URL env var not set')
 if (!RPC_API_KEY) throw new Error('RPC_API_KEY env var not set')
+if (!INDEXER_URL) throw new Error('INDEXER_URL env var not set')
 
 export const logger = new Logger({
   namespace: ['unchained', 'coinstacks', 'solana', 'api'],
   level: process.env.LOG_LEVEL,
 })
 
 const heliusSdk = new Helius(RPC_API_KEY)
+heliusSdk.connection.getParsedTransactions
```

Comment:
doesn't seem like we need this?
```suggestion
```

Reply:
😴 

fml

Fixed in [774cde9](https://github.com/shapeshift/unchained/pull/1052/commits/774cde9191e9bca28050cd3c10aac6bd998b7721)